recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
Technically the build number need not change as the recipe/build hasn't changed. That said, it is not a big deal to change it.